jujugui/static/gui/src/app/components/env-list/env-list.js

```diff
@@ -101,7 +103,12 @@ YUI.add('env-list', function() {
         <juju.components.Panel
           instanceName="env-list-panel"
           visible={true}>
-          <ul className="env-list">
+          <ul className="env-list"
+            role="menubar"
+            id="environmentSwitcherMenu"
+            aria-expanded="true"
+            aria-hidden="false"
```

Comment:
Yea I did but then thought. There is no harm in adding it as it can only aid the reader.